alerting/src/main/kotlin/org/opensearch/alerting/script/DocumentLevelTriggerExecutionContext.kt

```diff
@@ -25,10 +27,11 @@ data class DocumentLevelTriggerExecutionContext(
     constructor(
         monitor: Monitor,
         trigger: DocumentLevelTrigger,
+        monitorRunResult: MonitorRunResult<DocumentLevelTriggerRunResult>,
         alerts: List<Alert> = listOf()
     ) : this(
-        monitor, trigger, emptyList(), Instant.now(), Instant.now(),
-        alerts, emptyList(), emptyList(), null
+        monitor, trigger, monitorRunResult.inputResults.results, monitorRunResult.periodStart, monitorRunResult.periodEnd,
+        alerts, emptyList(), emptyList(), monitorRunResult.scriptContextError(trigger)
```

Comment:
Do we need to have a test to surface out the ``inputResults.error``?

Reply:
That is a good point, we should check if we get the expected data from there as well.

Reply:
I believe we should have a follow up PR or in this PR to have tests to verify every part of the `ctx`` values.